tests/MagentoHackathon/Composer/Magento/PluginTest.php

```diff
@@ -107,6 +110,65 @@ public function testDebugListenerIsAttached()
         $this->plugin->activate($this->composer, $this->io);
     }
 
+    public function testOnlyMagentoModulePackagesArePassedToModuleManager()
+    {
+        $this->composer->setPackage($this->createRootPackage());
+        $this->plugin->activate($this->composer, $this->io);
+        $moduleManagerMock = $this->getMockBuilder('\MagentoHackathon\Composer\Magento\ModuleManager')
+            ->disableOriginalConstructor()
+            ->setMethods(['updateInstalledPackages'])
+            ->getMock();
+
+        $refObject   = new ReflectionObject($this->plugin);
+        $refProperty = $refObject->getProperty('moduleManager');
+        $refProperty->setAccessible(true);
+        $refProperty->setValue($this->plugin, $moduleManagerMock);
```

Comment:
Not really, we can't use the constructor in this class as `Composer` instantiates it for us.